### PR TITLE
[WIP] initial commit for workbench code-base

### PR DESCRIPTION
**Pull request by RafayAK**
This is the code base for a configurable and modular experiment workbench:

Current configurable features are:
- [x] Selected datasets from sklearn
- [x] Select size of base-dataset
- [x]  Amount of augmented data to add (Gaussian noise with oracle function)
- [x]  Number training iterations
- [x]  Define feed-forward network from config
- [ ]  plot decision boundary for a given trained model.
- [ ]  WandB integration.
- [ ]  Selection for the number of runs for aggregated results.
- [ ]  Output accuracy both holistically and class-wise, also F1 score.
- [ ]  Plot distribution of weights and biases.
- [ ]  CI Test cases

### CODE REVIEW

**Comment by RafayAK** [comment]
Closing in favor of 
https://github.com/alexhernandezgarcia/data-augmentation/tree/hydra-torch-workbench
